docs/ai/conceptual/embeddings.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Using Embeddings to Extend Your AI Model's Reach"
+description: "Learn how embeddings extend the limits and capabilities of AI models in prompt engineering in .NET."
+author: catbutler
+ms.topic: concept-article #Don't change.
+ms.date: 04/11/2024
+
+#customer intent: As a .NET developer, I want to understand how embeddings extend LLM limits and capabilities in .NET so that I have more options during prompt engineering and better outcomes for my AI apps.
+
+---
+
+# Embeddings in .NET
+
+This article explains how embeddings work in .NET.
+
+Embeddings are a way you can give semantic memories to an LLM. Essentially, embeddings are numeric representations of non-numeric data that preserve its semantic value. You can use embeddings to help an AI model understand the meaning of inputs so that it can perform comparisons and transformations, such as summarizing text or creating images from text descriptions.
+
+Summarizing text is especially useful for [prompt engineering](prompt-engineering-in-dot-net.md), because it gives you a way to use far fewer tokens to repesent a given text. You use examples in prompt engineering to show an LLM what to do. However, LLMs limit the number of tokens per input, which constrains how much text you can include in examples. If you try to include more than the limit in one input, the model will ignore some or all of that input. Some LLMs also feature quota systems, another reason to watch your token counts.
```

Comment:
Yep, it's content for an instruction. I think it's more primary content than supporting, because the model applies the instruction to primary content, so it's much likelier to be long than supporting content. I'll revise, thanks!

Reply:
revised